### ISSUE TITLE
Regression: sending out incorrectly formatted "udev" uevents on udevd worker timeouts

### ISSUE THREAD

**Issue by prajnoha** [systemd/systemd#11652]
There's a regression in code sending out `udev` uevent when a `udevd` worker timeout occurs. In this case, there should be a `udev` uevent sent out without  `USEC_INITIALIZED` key so all the listeners know that a timeout has just occurred.

The part of the code sending out `udev` uevents on `udevd` worker timeout is this:
https://github.com/systemd/systemd/blob/6d6308f6774b4c684de7f3aab12cb752c59d5e2f/src/udev/udevd.c#L1355

The regression seems to be caused by commit a3ce813697bcc1c4644e097a2f1cd0459326d6ee.

When such incorrectly formatted `udev` uevent is sent, the `udevadm monitor` reports:

```
[1] raw/~ # udevadm --debug monitor
monitor will print the received events for:
UDEV - the event which udev sends out after rule processing
KERNEL - the kernel uevent

KERNEL[1435.175340] change   /devices/pci0000:00/0000:00:0a.0/virtio4/host2/target2:0:0/2:0:0:0/block/sda (block)
sda: sd-device: Device created from strv or nulstr lacks devpath, subsystem, action or seqnum.
sd-device-monitor: Failed to create device from received message: Invalid argument
```

Of course, any other `udev` uevent listener doesn't receive proper uevent with environment as well.

(The code I tested is at commit 6d6308f6774b4c684de7f3aab12cb752c59d5e2f which still has the bug present, so this is still an intra-release bug, but would be great we have this fixed till v241 is out.)

**Comment by yuwata** [comment]
OK. I will take a look.

**Comment by yuwata** [comment]
Fix is waiting in #11655. PTAL.
